test/exnumerable_test.exs

```diff
@@ -2,30 +2,57 @@ defmodule ExnumeratorTest do
   use ExUnit.Case
 
   @values ["sent", "read", "received", "delivered"]
+  @values_as_atoms [:sent, :read, :received, :delivered]
 
   defmodule Message do
     use Exnumerator,
       values: ["sent", "read", "received", "delivered"]
   end
 
+  defmodule MessageAsAtoms do
```

Comment:
I suggest renaming the other module correspondingly to keep the consistency.

Reply:
done